addons/project/static/tests/tours/project_sharing_tour.js

```diff
@@ -21,7 +21,7 @@ const projectSharingSteps = [...stepUtils.goToAppSteps("project.menu_main_pm", '
     trigger: '.ui-autocomplete a.dropdown-item:contains("Georges")',
     in_modal: false,
 }, {
-    trigger: 'footer > button[name="action_share_record"]',
+    trigger: 'footer button[name="action_share_record"]',
```

Comment:
```suggestion
    trigger: 'footer > div > button[name="action_share_record"]',
```

The code is not tested.